seed/data_importer/models.py

```diff
@@ -113,7 +113,7 @@ class ImportRecord(NotDeletableModel):
     matching_done = models.BooleanField(default=False)
     is_imported_live = models.BooleanField(default=False)
     keep_missing_buildings = models.BooleanField(default=True)
-    status = models.IntegerField(default=0, choices=IMPORT_STATII)
+    status = models.IntegerField(default=0, choices=IMPORT_STATUSES)
```

Comment:
Thank you for fixing this!

Reply:
If you want to go the next level, then put the IMPORT_STATUSES inside the class that is referencing it.

Reply:
Boom.  Scoped.  55b8435